lib/kernel-manager.js

```diff
@@ -40,30 +40,29 @@ export default class KernelManager {
     const language = this.getLanguageFor(grammar);
     const kernel = this._runningKernels[language];
     delete this._runningKernels[language];
-    return __guard__(kernel, x => x.destroy());
+    __guard__(kernel, x => x.destroy());
   }
 
 
-  restartRunningKernelFor(grammar, onRestarted) {
+  restartRunningKernelFor(grammar, onRestarted = () => {}) {
```

Comment:
We could check for existence [here](https://github.com/lgeiger/hydrogen/blob/43e8ae80732fab70a8c37669626d0bdb8009518e/lib/kernel-manager.js#L51) If you prefere.